src/components/__tests__/SearchResults.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import SearchResults from '../SearchResults';
+import useSearchResults from '../useSearchResults';
+import { SEARCH_PARAM_QUERY } from '../../constants';
+
+jest.mock('../useSearchResults');
+jest.mock('react-router-dom', () => ({
+  Link: 'Link'
+}));
+
+describe('<SearchResults/>', () => {
+  const getSubject = () => shallow(<SearchResults />);
+  describe('when it does not receive any search hits', () => {
```

Comment:
personal preference would be just the `it` with like 'renders no list for empty search hits` or something. no need to change this.

Reply:
ya matter of preference really, I like reading the output as:
```
when it does not receive any search hits
  it does not render a list of results
```